src/components/Inputs/TextInput/TextInput.js

```diff
@@ -129,7 +123,6 @@ PrivateTextInput.PUBLIC_PROPS = {
   labelCopy: PropTypes.string.isRequired,
   validator: PropTypes.func,
   onBlur: PropTypes.func,
-  onChange: PropTypes.func,
```

Comment:
I'm bummed about this. I suppose I could have left this and cut a branch off `master`, but since my changes directly affect this component I'm not able to spot check with full confidence without removing the props.onChange.

But now I have to worry that I've somehow caused a regression downstream. The intent of this whole PR should be risk free: expose the `restrict` method as a helper, make sure TextInput still works, pull it into monorepo and use the helper.

So, above said, since monorepo is using v1.0 (legacy DS) and doesn't even reference any of this code yet, and, CMS is pointed to the "quote crazy" branch for the time being, I suppose there's no risk 🤔 😞

Reply:
Confirming your thoughts on the overall status of monorepo/cms ds integration